dags/common/utils.py

```diff
@@ -254,6 +243,17 @@ def build_case_report_json(xml_path: str) -> dict:
     return case_report
 
 
+def validate_case_report(case_report: dict) -> bool:
```

Comment:
If this is the same as checking "if case_report.get('pmID')", is there some advantage to defining it?

Reply:
In case we need other validations. It only has 1 right now.